src/backgroundQueue/server.spec.ts

```diff
@@ -51,5 +56,47 @@ describe('makeQueueServer', () => {
       expect(response.statusCode).toBe(HTTP_STATUS_CODES.BAD_REQUEST);
       expect(response.json()).toHaveProperty('type', QueueProblemType.UNSUPPORTED_EVENT);
     });
+
+    describe('makeQueueServerPlugin', () => {
+      const setEnvVars = configureMockEnvVars(REQUIRED_ENV_VARS);
+      const mockFastify: FastifyInstance = {
+        addContentTypeParser: jest.fn(),
+        getDefaultJsonParser: jest.fn(),
+        get: jest.fn(),
+        post: jest.fn(),
+      } as any;
```

Comment:
Why don't we borrow the approach used elsewhere in this repo? We used all of those methods in other places and never had to mock Fastify to do so.

Reply:
Took an example from here.
https://github.com/relaycorp/veraid-authority/blob/main/src/utilities/fastify/plugins/jwksAuthentication.spec.ts

Reply:
But we're testing routes here.

Reply:
Mocking Fastify should only be done in exceptional circumstances (e.g., plugins), not when testing routes.